examples/ittapi/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2016-2020, Rahul Sheth, Ruslan Baratov
```

Comment:
extend to 2021 or maybe add an additional line with just Rahul Sheth, @rbsheth what do you think?

Reply:
Either way is okay for me 😄